lightkurve/tests/test_search.py

```diff
@@ -125,3 +127,17 @@ def test_source_confusion():
     desired_target = 6507433
     tpf = search_targetpixelfile(desired_target, quarter=8).download()
     assert tpf.targetid == desired_target
+
+def test_open():
+    k2_path = os.path.dirname(os.path.abspath(__file__)) + '/data/test-tpf-star.fits'
```

Comment:
I suggest replacing `os.path.dirname(os.path.abspath(__file__))` with `PACKAGEDIR`  (which is defined in `__init__.py`, i.e. you can do `from . import PACKAGEDIR`)